examples/material_gallery/lib/demo/progress_indicator_demo.dart

```diff
@@ -10,43 +10,48 @@ class ProgressIndicatorDemo extends StatefulWidget {
 }
 
 class _ProgressIndicatorDemoState extends State<ProgressIndicatorDemo> {
+  AnimationController _controller;
+  Animation<double> _animation;
 
   @override
   void initState() {
     super.initState();
-    controller = new AnimationController(
+    _controller = new AnimationController(
       duration: const Duration(milliseconds: 1500)
     )..forward();
 
-    animation = new CurvedAnimation(
-      parent: controller,
+    _animation = new CurvedAnimation(
+      parent: _controller,
       curve: new Interval(0.0, 0.9, curve: Curves.ease),
       reverseCurve: Curves.ease
     )..addStatusListener((AnimationStatus status) {
       if (status == AnimationStatus.dismissed)
-        controller.forward();
+        _controller.forward();
       else if (status == AnimationStatus.completed)
-        controller.reverse();
+        _controller.reverse();
     });
   }
 
-  Animation<double> animation;
-  AnimationController controller;
+  @override
+  void dispose() {
+    _controller.stop();
```

Comment:
for symmetry when disposing from a dispose() function, use dispose() rather than stop().